adafruit_connection_manager.py

```diff
@@ -111,11 +111,10 @@ def create_fake_ssl_context(
 
 
 def _get_radio_hash_key(radio):
-    class_name = radio.__class__.__name__
-    # trying to use wifi.radio as a key results in:
-    #   TypeError: unsupported type for __hash__: 'Radio'
-    # So just use the class name in this case
-    return class_name if class_name == "Radio" else radio
+    try:
+        return hash(radio)
+    except TypeError:
+        return radio.__class__.__name__
```

Comment:
You could just use `id(radio)`. Also do you care that this is going to work on 8.x.x?

Reply:
So, although we are moving people to 9.x, this shouldn't force someone to upgrade. So this allow it to work in all cases, and you can't have more then 1 `wifi.radio`, so using the name is fine.

I always worry about `id`, since we would store the reference and it could actually get garbage collected and then something else get that value (although probably would never happen, but I don't want to try to debug that in discord...)